deployment/volumes/neo4j-data.yaml

```diff
@@ -9,4 +9,4 @@
       - ReadWriteOnce
     resources:
       requests:
-        storage: {{ .Values.neo4jStorage }}
+        storage: 5Gi
```

Comment:
But why we have changed this here if we don’t merge? @mattwr18 
We haven’t used it for our DevOps at all.